connect-another-device/connect-another-device.js

```diff
@@ -4,15 +4,25 @@ module.exports = {
   name: 'Should the user see the "connect another device" screen',
   hypothesis: 'A nudge to connect another device will help increase multi-device users',
   startDate: '2015-01-01',
-  subjectAttributes: ['uniqueUserId', 'isMetricsEnabledValue', 'forceExperimentGroup'],
+  subjectAttributes: ['uniqueUserId', 'forceExperiment', 'forceExperimentGroup'],
   independentVariables: ['connectAnotherDevice'],
   eligibilityFunction: function (subject) {
-    if (subject) {
-      return !! (subject.forceExperimentGroup || subject.isMetricsEnabledValue);
+    if (! subject) {
+      return false;
     }
 
-    return false;
+    if (subject.forceExperiment === 'connectAnotherDevice') {
+      return true;
+    }
+
+    // Place 50% of users into the two buckets, meaning the feature
+    // will be enabled for ~25% of users. 'isMetricsEnabledValue' is
+    // ignored, we have sufficiently high numbers that we'll assume
+    // an equal number of users who have metrics enabled are in each
+    // bucket. This assumption is measured in DataDog.
+    return this.bernoulliTrial(0.5, subject.uniqueUserId);
   },
+
   groupingFunction: function (subject) {
     var GROUPS = ['control', 'treatment'];
```

Comment:
Yeah, still groups, I still want to run this as an AB test, Where A is the existing flow, B is the new flow.